src/areas/examples/inputs.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as classnames from 'classnames/bind';
+import { TextField, CheckboxField, ToggleField, RadioField, SelectField } from '@microsoft/azure-iot-ux-fluent-controls/lib/components/Field'
+import { DateTimeField } from '@microsoft/azure-iot-ux-fluent-controls/lib/components/DateTime'
+
+const cx = classnames.bind(require('./inputs.module.scss'));
+
+export function Inputs() {
+    const [textValue, changeTextValue] = React.useState('');
+    const [checkboxValue, changeCheckboxValue] = React.useState(true);
+    const [toggleValue, changeToggleValue] = React.useState(true);
+    const [radioValue, changeRadioValue] = React.useState('option1');
+    const [selectValue, changeSelectValue] = React.useState('option1');
+    const [dateTimeValue, changeDateTimeValue] = React.useState('');
+

```

Comment:
nit: would be nice to have an example for side effects, like for example how to do data loading for inputs

Reply:
added example for an async load

Reply:
the main readme points to the react docs, which features hooks pretty prominently now. Re: the area name, I didn't want to call it test because a lot of backend services have a test folder that contain actual test files - this area is more for leaning/playground.

Reply:
synced offline, keeping area name as examples.